Cannot compile reference-contracts with newest cdt 4.0.0

```
#0 25.00 -- Build files have been written to: /app/reference-contracts/build
#0 25.02 Scanning dependencies of target contracts_project
#0 25.03 [ 11%] Creating directories for 'contracts_project'
#0 25.09 [ 22%] No download step for 'contracts_project'
#0 25.11 [ 33%] No update step for 'contracts_project'
#0 25.11 [ 44%] No patch step for 'contracts_project'
#0 25.13 [ 55%] Performing configure step for 'contracts_project'
#0 25.14 -- Setting up CDT Wasm Toolchain 4.0.0 at /usr
#0 25.14 -- Setting up CDT Wasm Toolchain 4.0.0 at /usr
#0 25.19 -- The C compiler identification is Clang 9.0.1
#0 25.25 -- The CXX compiler identification is Clang 9.0.1
#0 25.26 -- Detecting C compiler ABI info
#0 25.30 -- Detecting C compiler ABI info - failed
#0 25.30 -- Detecting C compile features
#0 25.30 -- Detecting C compile features - done
#0 25.31 -- Detecting CXX compiler ABI info
#0 25.35 -- Detecting CXX compiler ABI info - failed
#0 25.35 -- Detecting CXX compile features
#0 25.36 -- Detecting CXX compile features - done
#0 25.36 CMake Error at CMakeLists.txt:29 (message):
#0 25.36   Found CDT version 4.0.0 but it does not satisfy version requirements:
#0 25.36   version '4.0.0' must have the same major version as the soft maximum
#0 25.36   version (3)
#0 25.36 
#0 25.36   Please use CDT version 3.0.x
#0 25.36 
#0 25.36 
#0 25.36 -- Configuring incomplete, errors occurred!
#0 25.36 See also "/app/reference-contracts/build/contracts/CMakeFiles/CMakeOutput.log".
#0 25.36 See also "/app/reference-contracts/build/contracts/CMakeFiles/CMakeError.log".

```

--- a/contracts/CMakeLists.txt
+++ b/contracts/CMakeLists.txt
@@ -11,7 +11,7 @@ option(SYSTEM_BLOCKCHAIN_PARAMETERS
 find_package(cdt)
 
 set(CDT_VERSION_MIN "3.0")
-set(CDT_VERSION_SOFT_MAX "3.0")
+set(CDT_VERSION_SOFT_MAX "4.0")
 # set(CDT_VERSION_HARD_MAX "")
 
 # Check the version of CDT

